hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -70,35 +85,69 @@ public void storeValidCertificate(BigInteger serialID,
   }
 
   @Override
-  public void revokeCertificate(BigInteger serialID) throws IOException {
+  public Optional<Long> revokeCertificates(
+      List<BigInteger> serialIDs,
+      X509CertificateHolder caCertificateHolder,
+      CRLReason reason,
+      CRLApprover crlApprover)
+      throws IOException {
+    Date now = new Date();
+    X509v2CRLBuilder builder =
+        new X509v2CRLBuilder(caCertificateHolder.getIssuer(), now);
+    List<X509Certificate> certsToRevoke = new ArrayList<>();
+    X509CRL crl;
+    Optional<Long> sequenceId = Optional.empty();
```

Comment:
Can you add some javadoc on the interface for the case where non of the certs are revoked due to either not found or already revoked? 
In this case, the caller need to check the returned Optional for empty.